CHANGELOG.md

```diff
@@ -18,6 +18,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Please consult the [Migration Guide](https://github.com/ExtraHorizon/javascript-sdk/blob/dev/MIGRATING_TO_V8.0.0.MD) for more information
 - New implementation of the hashing for oAuth1 signature generation
     - This change is not expected to have any impact on the SDK usage
+- **Breaking Change** For all unauthenticated methods the response will now undergo the same transformation steps as any other request.
```

Comment:
(silly) Can you add a `:` after  `breaking change`? Rendered, it looks pretty weird without.
Same for the `normalize custom keys` entry 🙏